account_invoice_facturx_buyer_ref/models/account_invoice.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Camptocamp
+# @author: Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    @api.model
+    def _cii_trade_agreement_buyer_ref(self, partner):
+        return (
+            partner.facturx_ref
+            or partner.commercial_partner_id.facturx_ref
+            or super().remove_facturx_xml_format_in_pdf_invoice(partner)
```

Comment:
I see "remove_facturx_xml_format_in_pdf_invoice" defined in hooks but not in the model

Reply:
OMG thanks !! I honestly don't know where that came from, wrong copy pasting I think :/

Fixed now :)